gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,32 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm_svc = PbmService.new(server, username, password) if @apiVersion >= '5.5'
```

Comment:
Correct, once we can the session cookie out of our existing vim connection we can remove [this](https://github.com/ManageIQ/manageiq/pull/9334/files/89e13dada2bfdffcb4314e55128b8a029660ac62#diff-bbf43957bba72bc96dbb08f2762c561dR6) completely